apps/api/src/domain/models/digital-text/page-identifier.ts

```diff
@@ -0,0 +1 @@
+export type PageIdentifier = string;
```

Comment:
Should we put this in a `types` directory?

Reply:
If the `types` directory is in the same dir as the entity, it would be consistent with the `errors` directory

Reply:
Done. Also note that I replaced some references to a duplicate definition in the `Book` directory as well.